packages/derive/src/lib.rs

```diff
@@ -93,6 +93,24 @@ impl Parse for Options {
 ///
 /// where `InstantiateMsg`, `ExecuteMsg`, and `QueryMsg` are contract defined
 /// types that implement `DeserializeOwned + JsonSchema`.
+///
+/// ## Set the version of the state of your contract
+///
+/// The VM will use this as a hint whether it needs to run the migrate function of your contract or not.
+///
+/// ```
+/// # use cosmwasm_std::{
+/// #     DepsMut, entry_point, Env,
+/// #     Response, StdResult,
+/// # };
+/// #
+/// # type MigrateMsg = ();
+/// #[entry_point]
+/// #[set_contract_state_version(2)]
```

Comment:
Isn't the term "contract" a bit unnecessary here (also in the section naming)? I mean the whole thing is the contract

Reply:
Renamed to `set_state_version`